tools/print-evaluated-rules.js

```diff
@@ -11,6 +14,19 @@ const { ESLint } = require('eslint');
     const configTypescriptTypechecked = await eslint.calculateConfigForFile(`${__dirname}/../test/typescript-type-checking/index.js`);
     const configAngular = await eslint.calculateConfigForFile(`${__dirname}/../test/angular/index.ts`);
 
+    const getDivergedRules = rules => (
+        Object.keys(rules)
+            .filter(key => !key.startsWith('@angular-eslint'))
+            .filter(key => !configTypescript.rules[key] && !configTypescriptTypechecked.rules[key])
+            .reduce((config, key) => {
+                config[key] = rules[key];
+                return config;
+            }, {})
+    );
+    const angularDivergedRules = Object.assign(getDivergedRules(angularRules, getDivergedRules(angularExtraRules)));
```

Comment:
Is this accurate? `getDivergedRules(angularRules, getDivergedRules(angularExtraRules))` is passing a second argument to getDivergedRules which does not seemed to be used by the function

Reply:
Fixed!